src/electron/main.tsx

```diff
@@ -16,6 +16,9 @@ import chiaConfig from '../util/config';
 import { i18n } from '../config/locales';
 import About from '../components/about/About';
 import packageJson from '../../package.json';
+import AppIcon from '../assets/img/chia64x64.png'
+
+const appIcon = nativeImage.createFromPath(__dirname + '/' + AppIcon);
```

Comment:
Do you know if the '/' path separator causes issues on Windows?

Reply:
Updated to `path.join(__dirname, AppIcon)` in the latest PR.